Language/Haskell/GhcMod/Logging.hs

```diff
@@ -78,6 +83,17 @@ gmLog level loc' doc = do
 
   gmlJournal (GhcModLog Nothing (Last Nothing) [(level, loc', msgDoc)])
 
+-- | Appends a collection of logs to the logging environment, with effects
+-- | if their log level specifies it should
+gmLog' :: (MonadIO m, GmLog m, GmOut m) => GhcModLog -> m ()
+gmLog' GhcModLog { gmLogMessages } = do
```

Comment:
We aren't in a rush are we? I have no problem with merging this I just usually like to finish reviews before merging PRs. The only reason I can think of would be to make collaborating on a branch easier but that doesn't apply since you don't have commit access to the main repo (yet ;).